mythril/ethereum/interface/leveldb/state.py

```diff
@@ -150,7 +150,7 @@ def get_and_cache_account(self, addr):
             rlpdata = self.trie.get(addr)
 
         if rlpdata != trie.BLANK_NODE:
-            o = rlp.decode(rlpdata, Account, db=self.db, address=addr)
+            o = rlp.decode(rlpdata, Account, db=self.db, addr=addr)
```

Comment:
Is this because of an update? 
If so I'm not seeing anything in our pipfile/requirements.txt fixing the version to the newer one.
(or is this backwards compatible?)

Reply:
Not exactly, this was intended to be `addr` for some reason was changed to `address`(in some previous refactor PRs)

Reply:
so this is actually a bug that is in mythril currently?

Reply:
yes.